src/images/gcloud_emulators/bigtable.rs

```diff
@@ -0,0 +1,100 @@
+use crate::{core::WaitFor, Image};
+use std::collections::HashMap;
+
+const CONTAINER_IDENTIFIER: &str = "google/cloud-sdk";
```

Comment:
Normally, we have one implementation of `Image` per docker image, not per application that is started from within the container.

I need to think about this but my initial tendency would be to only have one implementation of image and maybe `Args` be an enum with all the different things that one can start in that image?

Reply:
> I need to think about this but my initial tendency would be to only have one implementation of image and maybe Args be an enum with all the different things that one can start in that image?

Hmm, how we can then go about `ready_conditions()`, possibility of running an application with non-default `RunArgs` and so on?

Reply:
You could have different constructor functions on the struct that implements image and initialize `ready_conditions` as a field depending on what the user wanted to run.